examples/index.json

```diff
@@ -363,6 +375,10 @@
                 "name": "Check",
                 "url": "check"
             },
+            {
```

Comment:
Should we maybe move this after the panic BBE? The flow might be more natural that way right? Check first, then panic and check panic as the last one?

Reply:
Done